### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by d-oit** [abpframework/abp#6254] [Ideas]
A few translation are not perfectly translated / misspelling. 

Example - pull request following 
https://github.com/abpframework/abp/blob/dev/abp_io/AbpIoLocalization/AbpIoLocalization/Community/Localization/Resources/de-DE.json:

 "Done": "Ferig", -> **Fertig**

I love the idea from orchard. They use:

https://crowdin.com/project/orchard-core

Maybe also an option for Abp?

**Comment by KevinFarrow** [comment]
Looks good but unfortunately it's not open source

**Reply by d-oit** [reply]
Yes - but this free for open source projects like Orchard CMS. 
Maybe someone else knows another plattform for collaboration localization to discuss and review the localization. 

Example google translate english -> german: Columns = Säule in an data grid 

Localize "Excel User" end user use **Spalten** for **Columns** 

In my opion a good UI/UX needs a localized language.

**Comment by maliming** [comment]
hi @do-it-ger 

We have `abp translate` command to help.

https://github.com/abpframework/abp/pull/4088
https://docs.abp.io/en/abp/latest/CLI#translate

**Reply by d-oit** [reply]
@maliming Thanks for the reply. 
I read the cli command but haven't tried it out yet. For **Aspnetboilerplate** i wrote a non-generic tool for the translation. 
This also include the non-existing L("") strings into the resource file.  
Does the cli also create non existing L strings as an empty item?

_Back to the discussion_. I started the discussion after I found a misspelling in an existing de json translation file. 
No big deal for me to do this in github. I was thinking of end user or non github user with localization know-how. With any kind of easy editing localization the plattform / framework gets better.
